features/environment.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Git Town performs correctly depending on the environment
+
+  Scenario Outline: Git Town commands run outside of a Git repository
+    Given I'm currently not in a git repository
+    When I run `<COMMAND>` while allowing errors
+    Then I <DO_OR_DONT> see "This is not a git repository."
+    And I don't see "fatal: Not a git repository"
```

Comment:
Going with "error: This is not a git repository."

"Not inside" isn't quite as clear to me